config/settings.yml

```diff
@@ -1489,7 +1489,7 @@ virtual_regional_office:
   api_key: 9d3868d1-ec15-4889-8002-2bff1b50ba62
   evidence_pdf_path: evidence-pdf
   ctn_classification_path: classifier
-  max_cfi_path: max-ratings
+  max_cfi_path: employee-experience/max-ratings
```

Comment:
In case we wonder later why we decided to change this: This path is the same in all deployed envs, and only different locally. So it can be overriden locally using the settings.local.yml. See 🧵 in [slack](https://dsva.slack.com/archives/CBU0KDSB1/p1692900543267259).